test/basic.js

```diff
@@ -79,10 +79,10 @@ t.test('find when executable', function (t) {
     process.env.PATHEXT = '.SH'
 
     t.test('foo.sh', function (t) {
-      runTest('foo.sh', t)
+      runTest(fixture + '/foo.sh', t)
```

Comment:
My point is to test if the command can find an executable by it's relative path started from cwd.